apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/ManifestsList/CategoriesList/index.tsx

```diff
@@ -21,7 +22,13 @@ const renderItem = ({
 }) => {
   return (
     <Badge
-      onPress={() => extraData?.selectCategory(item.id)}
+      onPress={() => {
+        track("web3hub_category_clicked", {
+          component: `CategoriesList`,
+          category: item.id,
+        });
```

Comment:
Not sure about this one too as we should be able to track the category selected with a `TrackScreen` and the proper props
It would give us the guarantee that we actually updated the state when the user clicks instead of just tracking a click